components/status_dropdown/status_dropdown.jsx

```diff
@@ -17,12 +17,16 @@ import StatusAwayIcon from 'components/widgets/icons/status_away_icon';
 import StatusOnlineIcon from 'components/widgets/icons/status_online_icon';
 import StatusDndIcon from 'components/widgets/icons/status_dnd_icon';
 import StatusOfflineIcon from 'components/widgets/icons/status_offline_icon';
+import DndCustomTimePicker from 'components/dnd_custom_time_picker_modal/dnd_custom_time_picker_modal';
```

Comment:
Here is the problem, you are importing the component directly instead of the "connected component", you have to remove the last `dnd_custom_time_picker_modal` part.